editor/connections_dialog.cpp

```diff
@@ -872,7 +872,17 @@ ConnectDialog::~ConnectDialog() {
 
 Control *ConnectionsDockTree::make_custom_tooltip(const String &p_text) const {
 	// If it's not a doc tooltip, fallback to the default one.
-	return p_text.contains("::") ? nullptr : memnew(EditorHelpTooltip(p_text));
+	if (p_text.contains("::")) {
+		return nullptr;
+	}
+
+	EditorHelpBit *help_bit = memnew(EditorHelpBit(p_text));
+	EditorHoverableTooltip *tooltip = memnew(EditorHoverableTooltip(const_cast<ConnectionsDockTree *>(this)));
```

Comment:
You can avoid `const_casts` if you store the Control in a variable and then connect the signals in `_notification()`.

Reply:
I don't think this is possible. In any case you will need to remove the qualifier, since you cannot assign `const T *` to `T *`.

Reply:
Then maybe move the const cast to `EditorHoverableTooltip()` and make the argument const? It would at least make creating tooltips cleaner.

Reply:
I'd rather keep the `const_cast` in calls rather than move it into the definition. Because `EditorHelpBitTooltip` requires a non-const pointer for `connect()`. In my opinion, it is better not to hide the problem.